modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ObjectDefinitionNode/ObjectDefinitionNode.tsx

```diff
@@ -144,15 +144,38 @@ export function ObjectDefinitionNode({
 	};
 
 	const handleShowEditObjectDefinitionExternalReferenceCodeModal = () => {
+		const {edges, nodes} = store.getState();
+
 		setShowModal({
 			editObjectDefinitionExternalReferenceCode: true,
 		});
+
+		dispatch({
+			payload: {
+				draggable: false,
+				objectDefinitionNodes: nodes,
+				objectRelationshipEdges: edges,
+				selectedObjectDefinitionId: id.toString(),
+			},
+			type: TYPES.SET_SELECTED_OBJECT_DEFINITION_NODE_DRAGGRABLE,
+		});
 	};
 
 	const handleShowRedirectObjectDefinitionModal = () => {
+		const {edges, nodes} = store.getState();
 		setShowModal({
 			redirectToEditObjectDefinitionDetails: true,
 		});
+
+		dispatch({
+			payload: {
+				draggable: false,
+				objectDefinitionNodes: nodes,
+				objectRelationshipEdges: edges,
+				selectedObjectDefinitionId: id.toString(),
+			},
+			type: TYPES.SET_SELECTED_OBJECT_DEFINITION_NODE_DRAGGRABLE,
+		});
 	};
```

Comment:
Hey @Artartur, could you reorder this function and add a blank space after `const` declaration? i.e

```
const handleShowRedirectObjectDefinitionModal = () => {
  const {edges, nodes} = store.getState();

  dispatch({
    payload: {
      draggable: false,
      objectDefinitionNodes: nodes,
      objectRelationshipEdges: edges,
      selectedObjectDefinitionId: id.toString(),
    },
    type: TYPES.SET_SELECTED_OBJECT_DEFINITION_NODE_DRAGGRABLE,
  });

  setShowModal({
      redirectToEditObjectDefinitionDetails: true,
    });
  };
```

Reply:
done!